planning/behavior_velocity_planner/src/scene_module/blind_spot/scene.cpp

```diff
@@ -110,21 +109,32 @@ bool BlindSpotModule::modifyPathVelocity(
   }
   const size_t closest_idx = closest_idx_opt.get();
 
+  /* set judge line dist */
+  const double current_vel = planner_data_->current_velocity->twist.linear.x;
+  const double max_acc = planner_data_->max_stop_acceleration_threshold;
+  const double delay_response_time = planner_data_->delay_response_time;
+  const double pass_judge_line_dist =
+    planning_utils::calcJudgeLineDistWithAccLimit(current_vel, max_acc, delay_response_time);
+  const auto stop_point_pose = path->points.at(stop_line_idx).point.pose;
+  const auto distnce_until_stop = motion_utils::calcSignedArcLength(
+    input_path.points, current_pose.pose, stop_point_pose.position);
+  if (distnce_until_stop == boost::none) return true;
+
   /* get debug info */
   const auto stop_line_pose = planning_utils::getAheadPose(
     stop_line_idx, planner_data_->vehicle_info_.max_longitudinal_offset_m, *path);
   debug_data_.virtual_wall_pose = stop_line_pose;
-  debug_data_.stop_point_pose = path->points.at(stop_line_idx).point.pose;
-  debug_data_.judge_point_pose = path->points.at(pass_judge_line_idx).point.pose;
+  const auto stop_pose = path->points.at(stop_line_idx).point.pose;
```

Comment:
Isn't it same as the 'stop_line_pose'

Reply:
@purewater0901 
no it's stop pose for ego vehicle

Reply:
okay. got it